### PR TITLE
Updating humans.txt

### PR DESCRIPTION
**Pull request by NataliaLKB**
## What does this change?
This is pretty out of date! Lets update it a bit shall we?

### CODE REVIEW

File: facia/public/humans.txt [hunk 1]
```diff
@@ -106,10 +89,16 @@ Client-Side Developer: Simon Adcock
 Twitter: @siadcock
 Location: London, UK
 
+Client-Side Developer: George Haberis
+Location: London, UK
+
+Developer: Phil McMahon
```
Comment (NataliaLKB) [inline facia/public/humans.txt]: @philmcmahon please update this as you see fit!

File: facia/public/humans.txt [hunk 2]
```diff
@@ -106,10 +89,16 @@ Client-Side Developer: Simon Adcock
 Twitter: @siadcock
 Location: London, UK
 
+Client-Side Developer: George Haberis
```
Comment (NataliaLKB) [inline facia/public/humans.txt]: Please update this as you see fit @GHaberis !

**Comment by SiAdcock** [comment]
Could probably move Anne Byrne to former members too 😢

**Comment by NataliaLKB** [comment]
Good point @SiAdcock ! Done.

**Comment by paperboyo** [comment]
Super-small:
`Poznan` should be spelled `Poznań` (sorry, can’t add commits to a PR).

**Comment by philmcmahon** [comment]
Updated!

**Review by SiAdcock — APPROVED** [review]
❤️

**Pull request merged**